Dockerfile

```diff
@@ -1,39 +1,58 @@
-FROM alpine:latest
+FROM centos:latest
 LABEL maintainer="OSC"
 
 # Set language to avoid bugs that sometimes appear
 ENV LANG en_US.UTF-8
 
 # Set up requirements
-RUN apk upgrade --update \
-    && apk --no-cache add \
-          python3 \
-          py3-pip \
-          make \
-          openjdk8-jre \
-          ttf-dejavu \
-          graphviz
+RUN curl -sL https://rpm.nodesource.com/setup_10.x | bash - \
+    && yum -y install \	   
+       openjdk8-jre \
+       ttf-dejavu \
+       graphviz \
+       alsa-lib \
+       gtk3 \
+       libXScrnSaver \
+       libappindicator \
+       libnotify \
+       git \
+       wget \
+       openssl \
+       nodejs \
+       npm \
+       which \
+       gcc-c++ make \
+       epel-release \
+    && yum clean all \
+    && rm -rf /var/cache/yum
 
 # Install PlantUML
-RUN apk add --no-cache --virtual .ssl-deps \
-      openssl \
-      ca-certificates \
-    && mkdir -p /opt \
+RUN mkdir -p /opt \
     && wget -O "/opt/plantuml.jar" "https://sourceforge.net/projects/plantuml/files/plantuml.jar" \
     && printf '#!/bin/sh -e\njava -jar /opt/plantuml.jar "$@"' > /usr/local/bin/plantuml \
-    && chmod 755 /usr/local/bin/plantuml \
-    && apk del .ssl-deps
+    && chmod 755 /usr/local/bin/plantuml 
 
 # Install Sphinx and extras
 ADD Pipfile* /tmp/
 
-RUN pip3 install pipenv=='2018.11.26' \
-  && cd /tmp \
-  && pipenv install --deploy --system \
-  && rm -rf /tmp/Pipfile*
+# Install pipenv
+RUN yum install -y python36-pip.noarch \
+    && pip3 install pipenv=='2018.11.26' \
+    && cd /tmp \
+    && pipenv install --deploy --system \ 
+    && rm -rf /tmp/Pipfile* \
+    && yum clean all \
+    && rm -rf /var/cache/yum
+
+# Install drawio-batch
+RUN git clone "https://github.com/languitar/drawio-batch.git" \
+    && cd drawio-batch && npm -g install && npm install \
```

Comment:
Why do you run both `npm -g install` and `npm install`? Doesn't this result in installing two sets of the same dependencies?

Reply:
Also, I wonder if you specified `--production` if that would have any impact in the npm install step.

Reply:
I ran both commands mainly because I was having issues installing npm globally, mainly the issue was that chromium was not getting installed so I had to install locally, to install chromium, and globally to use the drawio-batch command. I have not used the `--production` specification.

Reply:
If you omit npm -g install you will still have a binary produced, it will just be in node_modules/.bin/ so you could just add that directory to the PATH. Speaking of which I don't know how you manipulate PATH for a user running Docker commands, @MorganRodgers?